sdk/identity/identity/src/credentials/managedIdentityCredential/tokenExchangeMsi.ts

```diff
@@ -2,24 +2,33 @@
 // Licensed under the MIT license.
 
 import fs from "fs";
-import { createHttpHeaders, PipelineRequestOptions } from "@azure/core-rest-pipeline";
+import {
+  createHttpHeaders,
+  createPipelineRequest,
+  PipelineRequestOptions
+} from "@azure/core-rest-pipeline";
 import { AccessToken, GetTokenOptions } from "@azure/core-auth";
 import { promisify } from "util";
 import { credentialLogger } from "../../util/logging";
 import { MSI, MSIConfiguration } from "./models";
-import { msiGenericGetToken } from "./utils";
 import { DefaultAuthorityHost } from "../../constants";
 
 const msiName = "ManagedIdentityCredential - Token Exchange";
 const logger = credentialLogger(msiName);
 
 const readFileAsync = promisify(fs.readFile);
 
+/**
+ * Formats the expiration date of the received token into the number of milliseconds between that date and midnight, January 1, 1970.
+ */
 function expiresInParser(requestBody: any): number {
   // Parses a string representation of the seconds since epoch into a number value
   return Number(requestBody.expires_on);
 }
```

Comment:
Same question about whether or not this abstraction is needed and whether or not we can give `requestBody` a better type.